include/dfu/pcd.h

```diff
@@ -69,6 +69,22 @@ enum pcd_status {
  */
 int pcd_network_core_update(const void *src_addr, size_t len);
 
+/** @brief Sets up the PCD command structure with the location and size of the
+ *	   firmware update. Then boots the network core and initiates the
+ *	   update procedure. The function will return before the update
+ *	   procedure on the network core has completed or failed, and will
+ *	   indicate succes as long as the initialization of the update
+ *	   procedure was successful. The network core will still be in a
+ *	   powered on state when this function returns.
+ *
+ * @param src_addr Start address of the data which is to be copied into the
+ *                 network core.
+ * @param len Length of the data which is to be copied into the network core.
+ *
+ * @retval 0 on success, an error code on failure.
+ */
+int pcd_network_core_update_initiate(const void *src_addr, size_t len);
```

Comment:
User should be informed that he must wait for some additional time after the progress bar reaches the end. Otherwise network core update would fail. We could suggest to update network core first in serial recovery too.

Reply:
@MarekPieta We can do that, but rather in smp_svr sample documentation. The mcumgr-cli is at this point rather test/debug tool and will, probably, be only used in development.
Regarding the order of uploading images, I agree that uploading netcore first may be better as it should allow the image to settle, while the application will be uploaded. It would also allow to check, in the future, whether netcore upload was successful before setting for swap the app image.